appinventor/appengine/src/com/google/appinventor/client/DesignToolbar.java

```diff
@@ -302,6 +302,11 @@ public void execute() {
     }
   }
 
+ public void switchToBlocksEditor() {
```

Comment:
I spent a lot of time in the code, and I can't figure out why we needed these new functions. It looks like Joanie wrote them. I removed them, and references to them, and the code still builds, the walkthrough tutorials still work. I don't think we need them.